tests/parser-test-files/parse-expression.txt

```diff
@@ -0,0 +1,299 @@
+    ((Some x) x)
+    (y 1)
+    (1 z)
+    (_ 2)))
+
+================================================================================
+Parse expression (:disable)
+================================================================================
+
+(package coalton-unit-tests)
+
+(define (g x)
+  (match x))
+
+================================================================================
+Parse expression (:disable)
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define (f)
+  (let x = 1)
+  (let (Some y) = 2)
+  (+ x y))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define f
+  (the String "value"))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests)
+
+(define (x) (return))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests)
+
+(define (y) (return 5))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define z (+ 1 (+ 2 (* 3 5))))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define (f)
+  (progn
+    (let x = 1)
+    (f x)
+    (let y = 2)
+    (+ x y)))
+
+================================================================================
+Parse expression
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define (f x y)
+  (if (== x y)
+    1
+    2))
+
+================================================================================
+Parse expression (:disable)
+================================================================================
+
+(package coalton-unit-tests/prelude
+  (import coalton-prelude))
+
+(define (f x y)
+  (when (== x y)
+    (let z = 3)
+    (print "hello"))
+  (unless (== x y)
+    (let z = 4)
+    (print "hello")))
+
+================================================================================
+Parse expression (:disable)
+================================================================================
+
+(package coalton-unit-tests)
+
+(define x (and 1 2 3))
```

Comment:
This test case should just be removed, unless it should be:
```
(define x (and True True True))
```

Reply:
removed